### PR TITLE
[DSS-156] Tag - Image Variant

### PR DESCRIPTION
**Pull request by anechol**
## Description

Adds image variation for the Tag component.

## Screenshots

Screen Shot 2022-09-28 at 2 37 49 PM

### Rails
- View [Tag](http://localhost:4000/pages/component/tag?tab=preview)
- Check that component matches Figma specs

### React
- View [Tag](http://localhost:4100/?path=/docs/sage-tag--default)
- Check that component matches Figma specs

## Testing in `kajabi-products`

(**LOW**) Adds image variation for the Tag component.

## Related

[DSS-156](https://kajabi.atlassian.net/browse/DSS-156)

### CODE REVIEW

**Review by pixelflips — CHANGES_REQUESTED** [review]

File: packages/sage-react/lib/Tag/Tag.jsx [hunk 1]
```diff
@@ -16,6 +17,7 @@ export const Tag = ({
     <span
       className={classNames}
```
Comment (philschanely) [inline packages/sage-react/lib/Tag/Tag.jsx:18]: Is the new root tag modifier missing in `classnames(...)` above as well?

**Review by philschanely — COMMENTED** [review]
A few suggestions here, but looking good!

**Comment by anechol** [comment]
Thanks @philschanely @pixelflips, I've addressed all the comments and stuff I've missed.

File: docs/app/views/examples/components/tag/_preview.html.erb [hunk 2]
```diff
@@ -8,4 +8,14 @@
 			"data-js-copy-button": "Label"
 		}
 	} %>
+
+	<h3 class="t-sage-heading-6">Image Variant</h3>
+	<%= sage_component SageTag, {
+		value: "Label",
+		image: "avatar/koray_okumus.png",
```
Comment (pixelflips) [inline docs/app/views/examples/components/tag/_preview.html.erb:15]: Should we update the properties tab to document the new image?

**Review by pixelflips — APPROVED** [review]
One minor comment around documentation, but otherwise LGTM! Nice work! 👍🏼

File: docs/app/views/examples/components/tag/_preview.html.erb (hunk 2 above)
Comment (anechol) [reply docs/app/views/examples/components/tag/_preview.html.erb:15]: Done!

**Review by philschanely — APPROVED** [review]
Looks great! 🙌

**Review by cameronsimony — APPROVED** [review]
👍

**Pull request merged**